index.html

```diff
@@ -0,0 +1,248 @@
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_03.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_04.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_05.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div>
+            <div class="speaker-box">
+                <img src="assets/speakers/speaker_06.png" alt="speaker1-img" class="speaker-image">
+                <div class="speaker-desc-box">
+                    <h4 class="speaker-name">Igor</h4>
+                    <p class="speaker-title">Madagastar Esports Arena, West-Hills Mall, Weija-Accra</p>
+                    <p class="speaker-desc">As President of the Ghana Esports Federation, Igor has coordinated many Esports events from the grass-root to the top of the ladder. He is known to have the biggest collection of top players in a single tournament.</p>
+                </div>
+            </div> -->
```

Comment:
- Please remove these deleted code to make your work clean

Reply:
All done.